diesel/src/macros/associations/belongs_to.rs

```diff
@@ -45,15 +45,25 @@
 ///
 /// [custom_derive]: https://crates.io/crates/custom_derive
 ///
-/// ```ignore
+/// ```no_run
+/// # #[macro_use] extern crate diesel;
+/// # #[macro_use] extern crate custom_derive;
+/// # table! { users { id -> Integer, } }
+/// # table! { posts { id -> Integer, user_id -> Integer, } }
+/// # custom_derive!{
+/// #     #[derive(Identifiable(users))]
+/// #     pub struct User {
+/// #         id: i32,
+/// #     }
+/// # }
 /// custom_derive! {
-///     #[derive(BelongsTo(User, foreign_key = user_id)]
-///     #[table_name(posts)]
+///     #[derive(BelongsTo(User, foreign_key = user_id, table = posts), Identifiable(posts))]
```

Comment:
No. I'm inclined to just drop `custom-derive` from the documentation in this case. I'm not happy with this change to the API. This push was also before the macros 1.1 RFC which changes things a lot now that it's landed. Core team is discussing what we want to do.